### PR TITLE
added definitions and changed header/title for aggregate results

### PR DESCRIPTION
**Pull request by kaitlyn-sharo**

### CODE REVIEW

File: dashboard-ui/src/components/AggregateResults/Metrics Eval Variables.xlsx
Comment (brianpippin) [inline dashboard-ui/src/components/AggregateResults/Metrics Eval Variables.xlsx:1]: I think we want to remove the spaces from the file names, I don't think Windows machines will like that.

File: dashboard-ui/src/components/AggregateResults/Metrics Evaluation Human Dataset Variables Dashboard.docx
Comment (brianpippin) [inline dashboard-ui/src/components/AggregateResults/Metrics Evaluation Human Dataset Variables Dashboard.docx:1]: Same comment as above.

File: dashboard-ui/src/components/AggregateResults/Metrics Eval Variables.xlsx
Comment (kaitlyn-sharo) [reply dashboard-ui/src/components/AggregateResults/Metrics Eval Variables.xlsx:1]: Updated

**Review by brianpippin — APPROVED** [review]
This looks good, Jennifer wanted to double check a couple of columns to make sure they look good, you should drop the export in the chat for her.

**Review by dereknop — APPROVED** [review]

**Pull request merged**